rest-server/src/controllers/job.js

```diff
@@ -114,5 +122,55 @@ const remove = (req, res) => {
   });
 };
 
+
+/**
+ * Get job config json string.
+ */
+const getConfig = (req, res) => {
+  Job.prototype.getJobConfig(
+    req.job.jobStatus.username,
+    req.job.name,
+    (configJsonString, error) => {
+      if (error === null) {
+        return res.status(200).json(configJsonString);
+      } else {
+        return res.status(500).json({
+          error: 'InternalServerError',
+          message: error.message,
+        });
+      }
+    }
+  );
+};
+
+/**
+ * Get job SSH info.
+ */
+const getSshInfo = (req, res) => {
```

Comment:
>req [](start = 20, length = 3)

should we validate "req" in routes when get ssh?

Reply:
No need. Because 'req' does not include any input data in JSON format. Now param.validate only validates job config JSON string.